mypy.ini

```diff
@@ -0,0 +1,12 @@
+[mypy]
+files = docs,share,src,tests
```

Comment:
Would this work?
```suggestion
files =
  docs,
  share,
  src,
  tests,
```

Reply:
let's try it, would make for a better diff

Reply:
This is not only about the diff. A comma-separated list without any whitespaces is rather hard to consume and it looks ugly visually. Making a list with items being distinct on separate lines helps the list to be seen as a list and not as one huge string. Whitespaces after commas would help a bit too, of course, but I feel like this extra step makes it even better :)

Reply:
The comment "would make for a better diff" did not mean to suggest there were not other reasons to do it, I was simply highlighting one of many, that I find important.